doc/nimc.rst

```diff
@@ -102,6 +102,7 @@ Source                           The source line that triggered a diagnostic
                                  message.
 StackTrace
 Success, SuccessX                Successful compilation of a library or a binary.
+BuildMode                        Kind of build: debug, release, danger
```

Comment:
pre-existing problem so ignore for this PR but in future work we should have a single source of truth/documentation and keep it DRY; they're already out of sync.